mobile/fixed_words.asm

```diff
@@ -339,9 +339,9 @@ Function11c1b9:
 
 Function11c254:
 	push af
-	ld a, BANK(s4_a007)
+	ld a, BANK(sEZChatMessages)
 	call OpenSRAM
-	ld hl, s4_a007
+	ld hl, sEZChatMessages
```

Comment:
Line 355 below this is `ld bc, 12` and should be `ld bc, EASY_CHAT_MESSAGE_LENGTH`.